### DISCUSSION TITLE
Which branch is the default?

### DISCUSSION THREAD

**Discussion by kristofk** [community/community#21879] [New to GitHub]
<p>Hi, I’m creating a big repo in which I create orphan branches with separate projects. Each branch has a README file.</p>
<p>However, when someone visits this repo I want to explain theme that each project is its own branch and what the idea of this big repo is as a whole. </p>
<p>For this, I’d like to create a central (or main) README file that appears by default. But there is no master branch.</p>
<p>So which branch does GitHub show by default or how can I make sure that the central README file appears first?:</p>
<ul>
<li>First one created</li>
<li>Alphabetical order by branch name</li>
<li>Can I select a custom one</li>
</ul>

**Comment by lee-dohm** [comment]
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/kristofk">@kristofk</a>,</p>
<p>You can go into your repository settings to <a href="https://help.github.com/articles/setting-the-default-branch/" rel="nofollow noopener">choose your default branch</a>.</p>
<p>Out of curiosity, is there a reason that you’re putting each project into an orphaned branch in the same repository as opposed to using separate repositories and linking them together with a repository that acts as a sort of index? I love to hear about how different people are using GitHub differently.</p>
<p>Thanks!</p>

**Comment by kristofk** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> I have one big repo for my small side projects. These side projects used only 1 branch and aren’t utilizing git heavily but there is a lot of them. So I don’t want a separate repo for each of these as I (and other people) can comprehend so many repoes in a list. Not to mention that these are usually unmaintaned projects.</p>
<p>Only projects that have a high impact, carry some kind of importance or are still maintained deserve a separate repo.</p>
<p>You can check out the repo that I have created this way <a href="https://github.com/kristofk/iOS-Side-Projects" rel="nofollow noopener">here</a>.</p>
<p>To be hones tough I haven’t even considered an “indexing” repo but even if I did I’d still use the current approach. This acts kind of like a folder for me. Keeps list of repos visually organized.</p>

**Comment by ThirdEyeRose** [comment]
<p>That’s really interesting. Thanks for sharing!</p>
